### ISSUE TITLE
Reopening: vscode.dev doesn't allow Ctrl+V clipboard usage after copying a file

### ISSUE THREAD

**Issue by EdricCantu** [microsoft/vscode#186097]
Closed reference issue: #185466
```
For whatever reason, Visual Studio Code for the web (vscode.dev) appears
to copy the path of a file when clicked on, and immediately copied. It is
a very helpful feature, but once I do it once, I am no longer able to copy
and paste anything else besides file paths and files.

I am forgetful, and I keep text selected on different files for later use
to keep track of my work. Sometimes I forget to focus my cursor to the
text editor, and I end up copying, using Ctrl + C, the file path; I know
because pasting into the text editor, using Ctrl+V on the text editor of a
file pastes the path of the file I previously was. I then go to manually
reselect the text from the said file whose path I'd copied, and paste the
text into the other, but it still gives me the same content, the file
path. The same filepath all the time, everywhere, until I copy another
file using Ctrl + C directly from the file selection menu, and then, only
then, does the in-vsc clipboard content change. it changes to that file's
path.

I can right click on the text editor to get the Paste option, And
successfully paste the desired clipboard content, but it requires browser
clipboard permissions, and I find two clicks of the right hand to be a
waste of time, even for 1 second. I can easily reload, and regain access
to my system clipboard (using Ctrl + V), but then I waste even more time
having to load the folder back into the site, and then giving two separate
read and write permissions.
```

Very sorry, but I seem to have let it slip by, after mjbvz (in the referenced issue) had said he fixed it. I believed him, but had not bothered to verify, because I believed I could avoid the mistake I made described in the above text. I did, for a while, until today. It still happens for me.
I was able to produce a gif of the occurrence as requested by mjbvz using the resource given by VSCodeTriageBot:

![rec](https://github.com/microsoft/vscode/assets/136949182/0d8a41a6-fe06-4679-b786-e7f64f6b13b6)

Version: 1.79.2
Commit: 695af097c7bd098fbf017ce3ac85e09bbc5dda06
User Agent: Mozilla/5.0 (X11; CrOS x86_64 14541.0.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
Embedder: vscode.dev

**Comment by mjbvz** [comment]
It's fixed in the insiders version of VS Code: 1.80. Please test on https://insiders.vscode.dev

**Comment by EdricCantu** [comment]
Thank you mjbvz. Sorry for the duplicate.
